WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanFragment.kt

```diff
@@ -66,8 +71,14 @@ class ScanFragment : Fragment(R.layout.scan_fragment) {
         viewModel.uiState.observe(
             viewLifecycleOwner,
             { uiState ->
-                if (uiState is Content) {
-                    refreshContentScreen(uiState)
+                when (uiState) {
```

Comment:
❓ Should we update visibility of the views so we don't draw multiple view over each other?

Reply:
We don't need to do that, because the RV is always empty when a loading/error fullscreen view is shown.